site/en/docs/extensions/mv3/getstarted/tut-focus-mode/index.md

```diff
@@ -0,0 +1,374 @@
+}
+```
+
+Great! Let's insert or remove the stylesheet using the [Scripting][api-scripting] API. Start by
+declaring the `"scripting"` permission in the manifest:
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  ...
+  "permissions": ["activeTab", "scripting"],
+  ...
+}
+```
+
+{% Aside 'success' %}
+
+The Scripting API does not trigger a [permission warning][doc-perms-warning].
+
+{% endAside %}
+
+Finally, in `background.js` add the following code to change the layout of the page:
+
+{% Label %}background.js:{% endLabel %}
+
+```js
+  ...
+      if (nextState === "ON") {
+        // Insert the CSS file when the user turns the extension on
+        await chrome.scripting.insertCSS({
+          files: ["focus-mode.css"],
+          target: { tabId: tab.id },
+        });
+      } else if (nextState === "OFF") {
+        // Remove the CSS file when the user turns the extension off
+        await chrome.scripting.removeCSS({
+          files: ["focus-mode.css"],
+          target: { tabId: tab.id },
+        });
+      }
+    }
+  });
+```
+
+{% Details %}
+{% DetailsSummary %}
+💡 **Can I use the Scripting API to inject code instead of a stylesheet?**
+{% endDetailsSummary %}
+
+Yes! You can use [`scripting.executeScript()`][api-scripting-es] to inject JavaScript. 
+
+{% endDetails %}
+
+### _Optional: Assign a keyboard shortcut_ {: #step-6 }
+
+Just for fun, let's add a shortcut to make it easier to enable or disable focus mode. Add the
+`“commands”` key to the manifest.
+
+{% Label %}manifest.json:{% endLabel %}
+
+```json
+{
+  ...
+  "commands": {
+    "_execute_action": {
+      "suggested_key": {
+        "default": "Ctrl+U",
+        "mac": "Command+U"
+      }
+    }
+  }
+}
+```
+
+The `“_execute_action”` key runs the same code as the `action.onClicked()` event, so no additional
+code is needed!
+
+## Test that it works {: #try-out }
+
+Verify that the file structure of your project looks like the following: 
+
+```text
+Focus mode/
+├── manifest.json
+├── background.js
+├── focus-mode.css
+└── images/
+    ├── icon-16.png
+    ├── icon-32.png
+    ├── icon-48.png
+    └── icon-128.png
+```
+
+### Load your extension locally {: #locally }
+
+To load an unpacked extension in developer mode, follow the steps in [Development
+Basics][doc-dev-basics-unpacked].
+
+### Open an extension documentation page {: #open-sites }
+
+Go to any of the following pages:
+
+- [Welcome to the Chrome Extension documentation][doc-welcome]
+- [Publish in the Chrome Web Store][cws-publish]
+- [Scripting API][api-scripting]
+
+Then, click on the extension action or press the keyboard shortcut `Ctrl + U` or `Cmd + U`.
+
+It should look like this:
+
+<figure>
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/6jSxXwVmjVv10NvwJ2Iu.png", 
+alt="Focus Mode extension in Welcome page", width="800", height="378", class="screenshot" %}
+  <figcaption>
+  Extension Welcome page with the Focus Mode extension enabled
+  </figcaption>
+</figure>
```

Comment:
The image currently shows both the reading time extension and focus mode extension running on the page.

1. I'd suggest either changing the image to only show the focus mode extension OR updating the description to say that the reading time extension is shown & include link to that tutorial.
2. To make it more obvious to readers what they should see, maybe we should include before and after screenshots that show the extension enabled and disabled?

Reply:
I like #2- the before and after approach. I'll test what that looks like.